src/components/Tooltip/Tooltip.st.css

```diff
@@ -93,3 +94,13 @@
     opacity: 0;
     transition: transform 80ms linear, applyOpacity 80ms linear;
 }
+
+.root:skin(error)::popoverContent {
+    background-color: #F64D43;
+    padding: 5px 12px;
+    border: 1px solid rgba(255,255,255,0.25);
```

Comment:
btw, I'm not sure the padding and the border are necessary here (I'll check with the designers), even though it looks pretty good in the storybook

Reply:
Ok. I'll wait for the answer.

Reply:
I've deleted padding.

Reply:
So the designers said we can also remove the border.